tests/PhrasesTest/ApplicationTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+namespace PhrasesTest;
+
+use Pdo;
+use Zend\Http\Request;
+use Zend\Http\Headers;
+use Phrases\Application;
+use Phrases\Entity\Phrase;
+use Zend\StdLib\Parameters;
+use Phrases\Persistence\MySQL;
+use PHPUnit_Framework_TestCase;
+use PhrasesTestAsset\ConsumedData;
+
+/**
+ * @huge
+ */
+class ApplicationTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Application
+     */
+    private $application;
+    /**
+     * @var \PDO
+     */
+    private $pdo;
+    private $persistence;
+
+    public function setUp()
+    {
+        $username = 'root';
+        $password = (getenv('CONTINUOUS_INTEGRATION') == 'true') ? '' : 'root';
+        $pdo = new Pdo('mysql:host=localhost', $username, $password);
+        $pdo->exec('CREATE DATABASE IF NOT EXISTS phrases_test');
+        unset($pdo);
+
+        $this->pdo = new Pdo('mysql:host=localhost;dbname=phrases_test', $username, $password);
+        $sql = 'CREATE TABLE IF NOT EXISTS phrases (
+            id INTEGER(11) PRIMARY KEY AUTO_INCREMENT,
+            title VARCHAR(255) NOT NULL UNIQUE,
+            text TEXT NOT NULL
+        ) Engine=InnoDB';
+
+        $this->pdo->exec($sql);
```

Comment:
@malukenho, what do you think to use the method [createTables](https://github.com/malukenho/phrases/pull/19/files#diff-b294ef2ee6c3f4ba61ef454ebe5ce813R7)?